### DISCUSSION TITLE
[FR] Fullscreen view mode

### DISCUSSION THREAD

**Discussion by chromer030** [damionx7/Zettel-Notes-Documentation#99] [Feature Requests]
It would be great to have an option to enable fullscreen in view mode.

Main purpose is having distraction free view with more screen coverage dedicated to note.

Mostly view mode is used for reviewing and fullscreen view is essential for it.

thanks for consideration.

**Comment by damionx7** [comment]
I have converted this issue to a feature request in discussions.
